### PR TITLE
Functioning subset of Oblique Strategies deck

### PR DESCRIPTION
**Pull request by sbenthall**
This PR implements a prototype version of an Oblique Strategies deck, see #40.

Several changes are included in this PR:
 - Data for 2 cards of the Oblique deck have 
 - The seed:deck script used to loop through the file directory for each deck. Since the Oblique deck has no images, this had to be changed. Now the script reads and loops through the CSV file for each deck.
 - The `images.js` service now has a `write()` method for printing text to an image (via a canvas buffer). This is used when there is no file available for a card

Together, these changes partially implement Oblique Strategies. Because of the core functionality changes to seed:deck, I want to submit this PR for review at this stage.

Future work to complete #40, not included in this PR, will involve supplying the complete deck data.
It will also require work on sizing and spacing the text properly in the `images.js` `write()` command.

### CODE REVIEW

File: app/discord/draw.js [hunk 1]
```diff
@@ -48,9 +48,16 @@ export default async function draw(message, opt) {
   }
 
   log.debug(`PGBG: ${message.author.username} drew the ${card.name} from ${opt} - ${card.image}`);
-    
   // build the message
-  const image = await images.draw(`./assets/decks/${opt.toLowerCase()}/${card.image}`, reversed ? 180 : 0);
+  var image;
```
Comment (benheller) [inline app/discord/draw.js:52]: `var` is seldom used in modern JS. `var` is only used when you need to declare a variable that's accessible in the parent function scope. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#scoping_rules_2 for a complete explanation.

`let` would be most appropriate here.

File: app/discord/draw.js [hunk 2]
```diff
@@ -48,9 +48,16 @@ export default async function draw(message, opt) {
...
+  if (card.file) {
+    image = await images.draw(
```
Comment (benheller) [inline app/discord/draw.js:54]: nit: stylistically, I would have put this all on a single line

File: bin/seed-db.js [hunk 3]
```diff
@@ -28,6 +28,10 @@ const decks = [{
   path: 'assets/decks/archetopics',
   name: 'Archetopics',
   description: 'Regenaissance Sigils'
+}, {
+  path: 'assets/decks/oblique',
+  name: 'Oblique',
+  description: 'Oblique Strategies by Brian Eno'
```
Comment (benheller) [inline bin/seed-db.js:34]: nit: it's actually by Brian Eno _and_ Peter Schmidt

**Review by benheller — APPROVED** [review]
If you don't mind, please address the minor nits, but otherwise looks great. Thank you for the addition!

File: app/discord/draw.js (hunk 2 above)
Comment (sbenthall) [reply app/discord/draw.js:54]: which lines into one line?

Comment (benheller) [reply app/discord/draw.js:54]: See the git diff on line `53` for the original, which has the entire assignment to `image =` on one line. This is purely personal preference, you can tell me to fuck off and merge it.

Comment (sbenthall) [reply app/discord/draw.js:54]: i see. i think in this case it goes over an 80 character per line soft rule in my editor.

**Comment by sbenthall** [comment]
This PR does not seem to be getting the new rebased commit in the original branch?

**Pull request merged**